MyApp/app/policies/story_policy.rb

```diff
@@ -21,19 +19,19 @@ def resolve
 
 
   def index?
-    true
+    user.deleted_at == nil
```

Comment:
its ok, but i think, if you want to prevent executing the action if user is deleted in the meantime, 
you should think about destroying the session after destroying user.

But  it shouldn't bothering you now. If you have time, investigate, it seems interesting.